src/main/java/com/katsudon/binder/service/CardService.java

```diff
@@ -13,4 +18,13 @@ public class CardService {
   public Card create(Card card) {
     return cardRepository.save(card);
   }
+
+  public List<Card> find(List<String> rarity, List<String> set, String name,
+                         String orderBy, String orderDir, int page, int pageSize) {
+    boolean ascending = orderDir.compareTo("asc") > 0;
```

Comment:
Im realizing that asc and desc might not be the variables to handle sorting.
We'll revisit this another time.
This does the job, but makes it so that any word > "asc" makes it descending. Kinda wonky imo but its fine. no need to change this